tests/e2e/c/hashing_contract.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// AUTOMATICALLY GENERATED. DO NOT EDIT!
+// Generated from hashing.sol by compile-contract.sh
```

Comment:
Can we add `compile-contract.sh` to this directory?

Reply:
We supported this until go-ethereum removed their Solidity compiler package. I'm not opposed to installing solc and using that where necessary.

I don't have a strong opinion on whether it's better to install solc and compile on-demand or hardcode and am open to either.